src/views/GovernanceOverview/index.tsx

```diff
@@ -40,9 +41,16 @@ const GovernanceOverview = () => {
             <i className="fa fa-angle-right margin-x-05" aria-hidden />
           </li>
           <li>
-            <Link to="/governance-overview" aria-current="location">
-              Add a new system or service
-            </Link>
+            <FlagToggle name="taskListLite">
```

Comment:
This is more of a question than feedback.

Would we be able to use the hook?

For example:
```
const flags = useFlag();

{flags.taskListLite ? (
    <Link to="/governance-task-list/new" aria-current="location">
        Add a new system or service
    </Link>
) : (
    <Link to="/governance-overview" aria-current="location">
        Add a new system or service
    </Link>
)}
```

Would something like this work? Is there something more advantageous about the `FlagToggle` component?

Reply:
You know, I'm not sure to be honest. I think we can probably get away with just `useFlags` and get rid of `FlagToggle`.